Automattic/wp-calypso
# build(docker): Set NODE_ENV when building the app for production

#### Changes proposed in this Pull Request

Set NOD_ENV when building the docker image. This is used to trigger the `env > production` config in [babel.config.js](https://github.com/Automattic/wp-calypso/blob/trunk/babel.config.js#L9), which removes propTypes.

Related to #49705

#### Testing instructions

* Smoke test live branch
* Using Chrome Dev Tools, search for `propTypes` and verify they have been removed from bundles.
